api/src/main/kotlin/eu/codeloop/thehub/jobs/sync/JobFetcher.kt

```diff
@@ -18,6 +19,7 @@ class JobFetcher(private val jonFetcher: JobApiFetcher, private val jobHtmlFetch
     private fun fetch(host: String, api: JobApiFetcher.Job): Job? {
         try {
             val html = jobHtmlFetcher.fetchDetails(host, api.key)
+            val approvedAt = OffsetDateTime.parse(api.approvedAt ?: "1970-01-01T00:00:00.000Z")
```

Comment:
Please use extract that date to `application.yml`. Please see my comment for `api/src/main/kotlin/eu/codeloop/thehub/jobs/model/JobEntity.kt`

Reply:
Done in `JobFetcher.kt`. I'm not sure how to do it in `JobEntity.kt`. See 68ffd046b55c3125900fb2003dace2d2be768f04 .